Classes/MSAIContextHelper.m

```diff
@@ -35,22 +35,22 @@ @implementation MSAIContextHelper {
 
 + (instancetype)sharedInstance {
   static MSAIContextHelper *sharedInstance = nil;
-  
```

Comment:
There seem to have been a couple of formatting differences. What should we do about those?